src/commands/apphosting-secrets-set.ts

```diff
@@ -39,7 +39,7 @@ export const command = new Command("apphosting:secrets:set <secretName>")
     'File path from which to read secret data. Set to "-" to read the secret data from stdin.',
   )
   .action(async (secretName: string, options: Options) => {
-    const howToAccess = `You can access the contents of the secret's latest value with ${clc.bold(`firebase apphosting:secrets:access ${secretName}`)}`;
+    const howToAccess = `You can access the contents of the secret's latest value with ${clc.bold(`firebase apphosting:secrets:access ${secretName}\n`)}`;
```

Comment:
I know this was already here but would prefer to inline these comments: see go/java-practices/constants#when-to-use-a-constant-versus-an-inlined-literal

Relevant passage:
"""
If, however, the value is used just once, or its meaning is self-explanatory in context, prefer inlining the literal. The indirection involved in using the constant makes its usage slightly less readable (when one wants to see the value), so it should be done only if it's worth it.
"""

This is the java style guide but it's widely applicable imo

Reply:
+1 on inlining, I was thinking the same.